aries-site/src/layouts/content/NotifTag.js

```diff
@@ -0,0 +1,18 @@
+import { Tag } from 'grommet';
+
+export const NotifTag = ({ size, color, allyDes, textVal }) => {
```

Comment:
```suggestion
export const NotificationTag = ({ size, color, allyDes, textVal }) => {
```

As a Grommet style, we tend to avoid abbreviation in our naming, erring on clarity of naming instead of brevity.